cukinia

```diff
@@ -653,6 +654,15 @@ _cukinia_i2c()
 	return $ret
 }
 
+# _cukinia_kversion
+# arg1: the kernel version
+_cukinia_kversion()
+{
+	local kversion="$1"
+	_cukinia_prepare "Checking if kernel version is $kversion"
+	test $(uname -r | cut -d "-" -f1 | rev | cut -d "." -f2- | rev) = $kversion
```

Comment:
Considering this only takes the first 2 (major.minor) version numbers in check, same as:
 `uname -r | cut -d. -f1,2` ?


`= $kversion`: _Double quote to prevent globbing and word splitting.shellcheck(SC2086)_
Pro tip: add Shellcheck to your editor, for example on VS Code it checks as you type.